dashboard/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.9 AS build
+FROM bitnami/node:14.4.0-prod AS build
```

Comment:
I needed to update beyond 8.9 because one of the packages required a later node. I wasn't sure why we weren't using bitnami/node, so just switched to that.

Reply:
btw, the latest LTS version is v12.18.1